recipes/service.rb

```diff
@@ -12,6 +12,7 @@
 service 'marathon' do
   provider   Chef::Provider::Service::Upstart
   supports   status: true, restart: true
-  subscribes :restart, 'template[marathon-init]'
+  subscribes :stop, 'template[marathon-init]'
+  subscribes :start, 'template[marathon-init]'
```

Comment:
When you subscribes to :stop and :start, do you have the garantee that the 'stop' will be executed before the 'start' ?